File: C_Programs/PrintTillInputNumberUsingDoWhileLoop.cpp
// PrintTillInputNumberUsingDoWhileLoop.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>

int main()
{
	int num, cnt = 1;
	printf("Enter the Number : ");
	scanf_s("%d", &num);
	do
	{
		printf("%d\t", cnt);
		cnt++;
	} while (cnt <= num);
	return 0;
}

File: Basic_Java_Programs/PrintTillInputNumberUsingDoWhileLoop.java
//print till inputed number using do while loop

import java.util.Scanner;

public class PrintTillInputNumberUsingDoWhileLoop {
    public static void main(String[] args){
        int num, cnt=1;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the Number : ");
        num=sc.nextInt();
        do {
            System.out.print(cnt+"\t");
            cnt++;
        } while (cnt <= num);
    }
}
